src/sdl2/video.c

```diff
@@ -232,7 +233,7 @@ void toggleFullscreen()
 
 void setFullscreenMode(uint8_t fullscreen)
 {
-    SDL_SetWindowFullscreen(gWindow, fullscreen ? SDL_WINDOW_FULLSCREEN : 0);
+    SDL_SetWindowFullscreen(gWindow, fullscreen ? SDL_WINDOW_FULLSCREEN_DESKTOP : 0);
```

Comment:
Is there any consequence of this in scaling or rendering or does it Just Work© in any resolution the user has?